### PR TITLE
DEVX-2114: Adds kafka-devops refs to existing Kubernetes demos

### PR DESCRIPTION
**Pull request by rspurgeon**
### Description 
Adds doc refs to existing K8s demos pointing to kafka-devops project

_What behavior does this PR change, and why?_
- [ ] Documentation

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._
- [ ] Documentation

### CODE REVIEW

File: kubernetes/docs/includes/base-demo/overview.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ Overview
 
 This demo shows a deployment of |cp| on |k8s-service-name-long| (|k8s-service-name|) leveraging |co-long| with mock data generation provided via the `Kafka Connect Datagen <https://www.confluent.io/hub/confluentinc/kafka-connect-datagen>`__.
 
+.. note:: See the :ref:`kafka_devops` project for an example of using Kubernetes in a production like environment targeting |ccloud|.
```
Comment (ybyzek) [inline kubernetes/docs/includes/base-demo/overview.rst]:
```suggestion
.. note:: See the :ref:`kafka_devops` project for an example of using Kubernetes in a production-like environment targeting |ccloud|.
```

File: kubernetes/docs/includes/replicator-cc-demo/overview.rst [hunk 2]
```diff
@@ -4,6 +4,8 @@ If you'd like a primer on running |co-long| in |k8s-service-name| with lower res
 
 This example is featured in the `Conquering Hybrid Cloud with Replicated Event-Driven Architectures blog post <https://www.confluent.io/blog/replicated-event-driven-architectures-for-hybrid-cloud-kafka/>`__ which provides more details on use cases for replicated event streaming architectures.
 
+.. note:: See the :ref:`kafka_devops` project for an example of using Kubernetes in a production like environment targeting |ccloud|.
```
Comment (ybyzek) [inline kubernetes/docs/includes/replicator-cc-demo/overview.rst]:
```suggestion
.. note:: See the :ref:`kafka_devops` project for an example of using Kubernetes in a production-like environment targeting |ccloud|.
```

**Review by ybyzek — COMMENTED** [review]
@rspurgeon these are exactly the types of additions that will help users discover `kafka-devops` project 👏 

There are other places in this examples repo that could use additions -- were you planning to use this PR or submit a different one?  Candidate rst files:

- ccloud/docs/ccloud-demos-overview.rst
- anything else in ccloud/docs/ ?

**Comment by rspurgeon** [comment]
> There are other places in this examples repo that could use additions -- were you planning to use this PR or submit a different one? Candidate rst files:
> 
>     * ccloud/docs/ccloud-demos-overview.rst
> 
>     * anything else in ccloud/docs/ ?

@ybyzek added links to ccloud demo overview.  If I see other appropriate spots while working with the demo in the future I'll file other changes.

**Comment by rspurgeon** [comment]
Updated for proper ref link to kafka-devops in docs-platorm

File: ccloud/docs/ccloud-demos-overview.rst [hunk 3]
```diff
@@ -118,6 +118,10 @@ You can choose between two examples:
 
 .. figure:: ../../kubernetes/replicator-gke-cc/docs/images/operator-demo-phase-2.png
 
+Kafka DevOps
+------------
+
+The :ref:`kafka-devops` project is a simulated production environment running a streaming application targeting |ak-long| on |ccloud|.  Applications and resources are managed by GitOps with a declarative infrastructure, Kubernetes, and the Operator Pattern.
```
Comment (ybyzek) [inline ccloud/docs/ccloud-demos-overview.rst]:
```suggestion
The :ref:`kafka-devops` project is a simulated production-environment running a streaming application targeting |ak-long| on |ccloud|.  Applications and resources are managed by GitOps with a declarative infrastructure, Kubernetes, and the Operator Pattern.
```

File: ccloud/docs/ccloud-demos-overview.rst [hunk 4]
```diff
@@ -118,6 +118,10 @@ You can choose between two examples:
 
 .. figure:: ../../kubernetes/replicator-gke-cc/docs/images/operator-demo-phase-2.png
 
+Kafka DevOps
+------------
```
Comment (ybyzek) [inline ccloud/docs/ccloud-demos-overview.rst:122]: Awesome.  Could you also please add a diagram?

File: ccloud/docs/ccloud-demos-overview.rst (hunk 3 above)
Comment (rspurgeon) [reply ccloud/docs/ccloud-demos-overview.rst]: this was added in a pushed commit

File: ccloud/docs/ccloud-demos-overview.rst (hunk 4 above)
Comment (rspurgeon) [reply ccloud/docs/ccloud-demos-overview.rst:122]: Diagram added and docs build tested locally against this branch

**Review by ybyzek — APPROVED** [review]
I have not built locally but LGTM 🎉

**Pull request merged**